evals/README.md

```
@@ -274,9 +279,34 @@ first-party class qualified name (`INHERITS` target), and that method overrides
 are attributed to the right base class (`OVERRIDES`).
 
 ```bash
-uv run python -m evals.inheritance --target codebase_rag
+uv run python -m evals.inheritance --target codebase_rag              # Python
+uv run python -m evals.inheritance --target repo --language java      # javac
+uv run python -m evals.inheritance --target repo --language cpp       # libclang
+uv run python -m evals.inheritance --target repo --language csharp    # Roslyn
 ```
 
+**Language scope, and why the numbers are not comparable across it** (issue
+#1190). Each arm grades a different unit, so each writes its own row label and a
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)